pkg/server/server.go

```diff
@@ -548,38 +549,38 @@ func (s *ProxyServer) serveRecvBackend(backend Backend, stream agent.AgentServic
 
 		case client.PacketType_DATA:
 			resp := pkt.GetData()
-			klog.Infof("<<< Received %d bytes of DATA from agentID %s, connID %d", len(resp.Data), agentID, resp.ConnectID)
+			klog.V(5).InfoS("Received data from agent", "bytes", len(resp.Data), "agentID", agentID, "connectionID", resp.ConnectID)
 			frontend, err := s.getFrontend(agentID, resp.ConnectID)
 			if err != nil {
-				klog.Warning(err)
+				klog.ErrorS(err, "could not get frontent client")
 				break
 			}
 			if err := frontend.send(pkt); err != nil {
-				klog.Warningf("<<< DATA send to client stream error: %v", err)
+				klog.ErrorS(err, "send to client stream failure")
```

Comment:
At some point we may want to log differently based on broken connection vs other errors.

Reply:
Any suggestions? The fact that's an error should be taken into account.

Reply:
Its likely to be a nested error and I'm not sure what the root error is. This was more a note for the future than a suggestion for this PR. Sort of placeholder for future work :+1: